classes/class_basematerial3d.rst

```diff
@@ -23,7 +23,7 @@ This provides a default material with a wide variety of rendering features and p
 Tutorials
 ---------
 
-- :doc:`../tutorials/3d/spatial_material`
+- :doc:`../tutorials/3d/standard_material_3d`
```

Comment:
For the reference, this should be fixed in https://github.com/godotengine/godot directly, but for now I'm just hacking it. I don't want to sync the classref with the master branch for the time being as all the node renames will require tons of changes to the docs, which are best done once the refactoring discussed in #3390 has been done. Until then the `master` branch of the docs can stay close to `3.2` and thus make it easy to merge improvements and fixes to the stable branch.